var/spack/repos/builtin/packages/py-xtb/package.py

```diff
@@ -25,3 +25,10 @@ class PyXtb(PythonPackage):
     depends_on("py-meson-python", type="build")
     depends_on("py-numpy", type=("build", "run"))
     depends_on("xtb", type=("build", "run"))
+
+    # https://github.com/grimme-lab/xtb-python/pull/114
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/grimme-lab/xtb-python/pull/114.patch?full_index=1",
```

Comment:
Now that the patch is merged, can we reference it from the commit hash instead?

Reply:
sure i'll try to update today.